agent/command/results_xunit_parser.go

```diff
@@ -106,6 +108,14 @@ func (tc testCase) toModelTestResultAndLog(conf *internal.TaskConfig) (testresul
 	// Replace spaces, dashes, etc. with underscores.
 	res.TestName = util.CleanForPath(res.TestName)
 
+	if math.IsNaN(float64(tc.Time)) {
+		logger.Task().Errorf("Test '%s' time was NaN, its calculated duration will be incorrect", res.TestName)
+	}
+	if math.IsInf(float64(tc.Time), 0) || math.IsInf(float64(tc.Time), -1) {
```

Comment:
Per [the docs](https://pkg.go.dev/math@go1.20.3#IsInf), it seems like checking `math.IsInf(float64(tc.Time), 0)` checks both positive and negative infinity, so it seems like the second `-1` check can be omitted.

Reply:
Ah yes, thanks